examples/trpo_bullet_kuka.py

```diff
@@ -0,0 +1,57 @@
+from rllab.algos.trpo import TRPO
+from rllab.baselines.linear_feature_baseline import LinearFeatureBaseline
+from rllab.envs.bullet_env import BulletEnv
+from rllab.envs.normalized_env import normalize
+from rllab.misc.instrument import run_experiment_lite
+from rllab.policies.gaussian_mlp_policy import GaussianMLPPolicy
+
+from rllab.misc.rllab_gym_register import Register
+
```

Comment:
Always format your code with YAPF using the PEP8 setting.

```sh
sudo pip install --upgrade yapf
cd rllab
git diff --name-only origin/integration | grep "\.py" | xargs yapf -ip -vv --style=pep8
git commit --amend
```

Reply:
Okay